WORKSPACE

```diff
@@ -220,6 +221,14 @@ pypi_repository(
     version = "3.4.0",
 )
 
+pypi_wheel_repository(
```

Comment:
I just thought, you should delete the `/third_party/tensorflow` then. And ensure everything is working fine.